### DISCUSSION TITLE
How to export otel logs/traces to splunk_hec in json format

### DISCUSSION THREAD

**Discussion by shravani154** [open-telemetry/opentelemetry-collector-contrib#29528] [Q&A]
When sending OpenTelemetry (OTel) logs to Splunk through the HTTP Event Collector (HEC) token, the logs are currently exported in their raw format. Is there a specific property available to facilitate exporting the logs in JSON format for splunk exporter like we have encoding property for kafka exporter?

**Comment by omegion** [comment]
I face the same problem. Container logs are ending up raw in Splunk.
